polaris-react/.storybook/polaris-readme-loader.js

```diff
@@ -3,7 +3,8 @@ const chalk = require('chalk');
 const grayMatter = require('gray-matter');
 const MdParser = require('./md-parser');
 const React = require('react');
-const lodash = require('lodash');
+const startCase = require('lodash.startcase');
```

Comment:
According to https://lodash.com/per-method-packages  the per-package lodash stuff is considered deprecated and its usage is discouraged. It think it'd be better to have a devDependency on `lodash` and import from `lodash/startCase` / `lodash/camelCase` in these dev dependency cases.